packages/core/utils/src/dml/__tests__/entity_builder.spec.ts

```diff
@@ -122,6 +122,56 @@ describe("Entity builder", () => {
     ])
   })
 
+  test("define an entity with default value", () => {
+    const model = new EntityBuilder()
+    const user = model.define("user", {
+      id: model.number(),
+      username: model.text().defaultsTo("foo"),
+      email: model.text(),
+    })
+
+    const User = createMikrORMEntity(user)
+    expectTypeOf(new User()).toMatchTypeOf<{
+      id: number
+      username: string
+      email: string
+    }>()
+
+    const metaData = MetadataStorage.getMetadataFromDecorator(User)
+    expect(metaData.className).toEqual("User")
+    expect(metaData.path).toEqual("User")
+    expect(metaData.properties).toEqual({
+      id: {
+        reference: "scalar",
+        type: "number",
+        columnType: "integer",
```

Comment:
Q: we use integer by default, should we allow to configure that?? maybe not now but later a user could easily want numeric or big int no?

Reply:
I do not get it. We have a schema type called `model.number` which maps to `integer` data-type. Do you want to call it `model.integer` instead?

Reply:
no no we can keep it as it, I was just thinking about the other "number" types 😂